jmh-core/src/main/java/org/openjdk/jmh/runner/Runner.java

```diff
@@ -830,9 +830,13 @@ private BenchmarkParams newBenchmarkParams(BenchmarkListEntry benchmark, ActionM
     private List<String> getPrintPropertiesCommand(String jvm) {
         List<String> command = new ArrayList<>();
 
-        // use supplied jvm, if given
+        // use supplied jvm
         command.add(jvm);
 
+        // Avoid VM warnings interspersing with XML, eg "Duplicate cpuset controllers detected"
+        command.add("-XX:+IgnoreUnrecognizedVMOptions");
+        command.add("-Xlog:all=error");
+
```

Comment:
I think pushing `-Xlog:` to unsuspecting VMs is asking for trouble, even with `IgnoreUnrecognizedVMOptions`. Let's not do this.